Bottom: a704d701863d9f9215e9818cc69b2eb7c3beb376
Top:    c3e0646d18a5e1b6cb0808c2c469a8c85aeda1ea
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-14 00:27:08 +0000

Refresh of remove-unused-prototypes-setzo

---

diff --git a/src/action.c b/src/action.c
index b29d40b..d0e0ca3 100644
--- a/src/action.c
+++ b/src/action.c
@@ -530,7 +530,7 @@ FinishStroke (void)
 	case 96521:
 	case 96541:
 	case 98541:
-	  SetZoom (1000);	/* special zoom extents */
+	  /* XXX: FIXME: Call a zoom-extents action */
 	  break;
 	case 159:
 	case 1269:
@@ -541,28 +541,8 @@ FinishStroke (void)
 	case 12569:
 	case 12589:
 	case 14589:
-	  {
-	    Coord x = (StrokeBox.X1 + StrokeBox.X2) / 2;
-	    Coord y = (StrokeBox.Y1 + StrokeBox.Y2) / 2;
-	    double z;
-	    /* XXX: PCB->MaxWidth and PCB->MaxHeight may be the wrong
-	     *      divisors below. The old code WAS broken, but this
-             *      replacement has not been tested for correctness.
-	     */
-	    z =
-	      1 +
-	      log (fabs (StrokeBox.X2 - StrokeBox.X1) / PCB->MaxWidth) /
-	      log (2.0);
-	    z =
-	      MAX (z,
-		   1 +
-		   log (fabs (StrokeBox.Y2 - StrokeBox.Y1) / PCB->MaxHeight) /
-		   log (2.0));
-	    SetZoom (z);
-
-	    CenterDisplay (x, y);
-	    break;
-	  }
+	  /* XXX: FIXME: Zoom to fit the box StrokeBox.[X1,Y1] - StrokeBox.[X2,Y2] */
+	  break;
 
 	default:
 	  Message (_("Unknown stroke %s\n"), msg);
diff --git a/src/set.h b/src/set.h
index d3b9a1f..9978619 100644
--- a/src/set.h
+++ b/src/set.h
@@ -34,7 +34,6 @@
 
 void SetTextScale (int);
 void SetGrid (Coord, bool);
-void SetZoom (double);
 void SetLineSize (Coord);
 void SetViaSize (Coord, bool);
 void SetViaDrillingHole (Coord, bool);
@@ -45,7 +44,6 @@ void SetMode (int);
 void SetCrosshairRangeToBuffer (void);
 void SetRouteStyle (char *);
 void SetLocalRef (Coord, Coord, bool);
-void RedrawZoom (Coord, Coord);
 void SaveMode (void);
 void RestoreMode (void);
 void pcb_use_route_style (RouteStyleType *);
